Documentation/Profiling/davbr-blog-archive/Attach.md

```diff
@@ -1,5 +1,7 @@
 *This blog post originally appeared on David Broman's blog on 11/4/2009*
 
+***This method is only for desktop and does not work for CoreCLR. Please see [Profiler Attach on CoreCLR](../Profiler Attach on Coreclr.md) for profiler attach on CoreCLR***
```

Comment:
To keep it clear what is the original archived content and what isn't I'd suggest putting this above the '*This blog post' line and phrase it like:

[Update 5/30/19]: The archived content below refers to an Attach mechanism that only worked on desktop, not on .Net Core. Please see ...